### PR TITLE
Home

### PR DESCRIPTION
**Pull request by Digambar1411**
Added home page

image
image

### CODE REVIEW

File: Home/home.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    padding: 12px 3rem;
+    background-color: var(--light-sky-blue);
+}
+
+.nav-icon{
+    gap:1rem;
+}
+
+.wrapper{
+    max-width: 700px;
```
Comment (Rohitprasad83) [inline Home/home.css:18]: avoid using pixel

**Review by Rohitprasad83 — APPROVED** [review]
change pixel to relative units like rem

File: Home/home.css [hunk 2]
```diff
@@ -0,0 +1,97 @@
+root{
+    --light-sky-blue:rgb(192, 210, 228);
+    --search-input-border:#e2e2e2;
+
+}
+.nav{
+    justify-content: space-between;
+    padding: 12px 3rem;
```
Comment (HeyNitin) [inline Home/home.css:9]: can use rem

File: Home/home.css [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    background-color: var(--light-sky-blue);
+}
+
+.nav-icon{
+    gap:1rem;
+}
+
+.wrapper{
+    max-width: 700px;
```
Comment (HeyNitin) [inline Home/home.css:18]: use rem

File: Home/home.css [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+  
+.search-btn {
+    border-radius: 50px;
+    border: 0;
+    font-size:1rem;
+   
+}  
+  
+.search-input {
+    width:200px;
```
Comment (HeyNitin) [inline Home/home.css:35]: rem

**Review by HeyNitin — CHANGES_REQUESTED** [review]
Use rem instead of pixels, rest looks good to me

File: Home/home.css (hunk 4 above)
Comment (Digambar1411) [reply Home/home.css:35]: thanks for review i will use of rem

File: Home/home.css (hunk 1 above)
Comment (Digambar1411) [reply Home/home.css:18]: yeah thanks for review i will use rem

**Review by vaibhav18matere — COMMENTED** [review]
add search icon instead of button in navbar

**Pull request merged**